packages/fiori/src/ShellBar.ts

```diff
@@ -693,35 +756,34 @@ class ShellBar extends UI5Element {
 					order: shouldStayOnScreen ? 1 : -1,
 				},
 			};
-		});
+		}) as Array<IShelBarItemInfo>;
```

Comment:
if we make callback of map to return IShelBarItemInfo we can avoid casting to `Array<IShelBarItemInfo>`

```ts
_getAllItems(!!hasIcons).map((info): IShelBarItemInfo => {
```